app/views/layouts/navbar/_browse_menu.html.erb

```diff
@@ -1,6 +1,6 @@
 <li class="dropdown" id="browse-menu">
   <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
-    <span class="glyphicon glyphicon-search" aria-hidden="true"></span>
+    <%# <span class="glyphicon glyphicon-search" aria-hidden="true"></span> %>
```

Comment:
is this intentional? The icon is now missing from the menu (both browser and mobile)